nebu/cli/get.py

```diff
@@ -53,13 +53,6 @@ def get(ctx, env, col_id, col_version, output_dir, book_tree, get_resources):
     if resp.status_code >= 400:
         raise MissingContent(col_id, req_version)
     col_metadata = resp.json()
-    if col_metadata['collated']:
```

Comment:
Does it make sense to "overcheck" here and see if it is still collated although non collated is requested?
I'm guessing this should also never happen. Ignore this comment if this is one "check" too much.

Reply:
That's the point of removing this. The "overcheck" here doesn't do anything for us. It adds confusion and a conditional code path that can be averted by explicitly asking for what is wanted in the first place.

I'd guess this was in place because requesting with the query string is not going to hit the cache. But I don't see that as a completely valid reason for coding ambiguity.

Reply:
Just thought about this too... The other portion of this is that Nebuchadnezzar is primarily used on OpenStax books. Most of those books are now baked books, so it's making a double request anyhow. Therefore, this most certainly makes more sense when thinking of it in those terms.

Reply:
ok :+1: